ts4.1/index.d.ts

```diff
@@ -234,12 +235,12 @@ export interface TFunction<N extends Namespace = DefaultNamespace, TKPrefix = un
   ): TFuncReturn<N, TKeys, TDefaultResult, TKPrefix>;
 }
 
-export interface TransProps<
+export type TransProps<
```

Comment:
It's better to use interface so people can augment `TransProps` if they need

Reply:
I changed it on purpose because I received the following error with an interface:
```
Property 'style' does not exist on type 'IntrinsicAttributes & TransProps<Normalize<IScreens>, "screens", undefined, HTMLProps<HTMLDivElement>>'.
```
I have not looked into what causes the error, because it worked with a `type`.
If this is a dealbreaker, I will put more research into it.
What exactly do you mean by `augment`?

Reply:
This is module augmentation https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation

Are you certain that this doesn't work? It should:
```ts
export interface TransProps<
  K extends TFuncKey<N, TKPrefix> extends infer A ? A : never,
  N extends Namespace = DefaultNamespace,
  TKPrefix = undefined,
  E extends React.HTMLProps<HTMLDivElement> | TextProps
> extends E {
  children?: React.ReactNode;
  components?: readonly React.ReactNode[] | { readonly [tagName: string]: React.ReactNode };
  count?: number;
  defaults?: string;
  i18n?: i18n;
  i18nKey?: K | K[];
  ns?: N;
  parent?: string | React.ComponentType<any> | null; // used in React.createElement if not null
  tOptions?: {};
  values?: {};
  shouldUnescape?: boolean;
  t?: TFunction<N, TKPrefix>;
}
```